README.md

```diff
@@ -41,59 +45,109 @@ We recommend that you use `@dhis2/ui` as the entrypoint for all imports of our f
 
 ### `@dhis2/ui-constants`
 
-This package provides access to shared constants, such as colors, spacers and elevation values. They are used across our frontend components and can be used directly in applications as well. Our constants can be imported like so:
+This package provides access to shared constants, such as colors,
+spacers and elevation values. They are used across our frontend
+components and can be used directly in applications as well. Our
+constants can be imported like so:
 
 ```js
 import { colors } from '@dhis2/ui'
 ```
 
-See our [api docs](https://ui.dhis2.nu/#/api) for a full list of the available constants.
+See our [api docs](https://ui.dhis2.nu/#/api) for a full list of the
+available constants.
+
+### `@dhis2/ui-elements`
+
+Think of these components as styled HTML elements. These are the
+elements that are used to compose other UI components.
+
+-   Elements MUST NOT import other elements.
+-   Elements SHOULD NOT provide behavior.
+-   Elements MUST provide relevant styles for the design system.
```

Comment:
This seems a little unclear to me. I think I'd find something like `Elements MUST provide relevant style primitives for the design system.` more clear. Though that is still open to interpretation: what is relevant? What is the definition of a style primitive (you could also call it an element, or a styled element, but still)?

Reply:
The other rules all seem a lot more concrete to me, less open to interpretation. Would be good if we could whittle this rule down to that level as well.